pkg/odo/cli/project/get.go

```diff
@@ -51,12 +53,14 @@ func (pgo *ProjectGetOptions) Validate() (err error) {
 
 // Run runs the project get command
 func (pgo *ProjectGetOptions) Run() (err error) {
-	project := pgo.Context.Project
+	curProject := pgo.Context.Project
```

Comment:
nitpick, but we should use full variables names. So `currentProject` is more suitable than `curProject`

Reply:
updated with recommendation, thanks~